files/zh-cn/mdn/contribute/markdown_in_mdn/index.md

```diff
@@ -238,12 +238,12 @@ const greeting = "I'm a bad example";
 
 - GFM `<ul>` 包含任意数量的顶级 GFM `<li>` 元素。
 - 每一个顶级 GFM `<li>` 元素必须包含一个 GFM `<ul>` 元素作为其最后一个元素。
-- 最内层的 `<ul>` 必须包含一个单独的 GFM `<li>` 元素，其文本内容必须以 `: `（冒号后跟空格）为开头。这个元素可能包含块级元素：包括段落、代码块、嵌套的列表和备注。
+- 最内层的 `<ul>` 必须包含一个单独的 GFM `<li>` 元素，其文本内容必须以 `:`（冒号后跟空格）为开头。这个元素可能包含块级元素：包括段落、代码块、嵌套的列表和备注。
```

Comment:
I think the original content is right here, this may raise warnings. So could we just modify like this:

```suggestion
- 最内层的 `<ul>` 必须包含一个单独的 GFM `<li>` 元素，其文本内容必须以“: ”（冒号后跟空格）为开头。这个元素可能包含块级元素：包括段落、代码块、嵌套的列表和备注。
```

Reply:
Thanks! Not sure if the common quote style is `": "` or the smart quotes in your suggestion `“: ”`

Reply:
using `“: ”` is ok for **Simplified Chinese**